src/openrct2/actions/GameAction.cpp

```diff
@@ -140,6 +140,57 @@ namespace GameActions
         return result;
     }
 
+    static const char* GetRealm()
+    {
+        if (network_get_mode() == NETWORK_MODE_CLIENT)
+            return "cl";
+        else if (network_get_mode() == NETWORK_MODE_SERVER)
+            return "sv";
+        return "";
+    }
+
+    struct ActionLogContext_t
+    {
+        MemoryStream output;
+    };
+
+    static void LogActionBegin(ActionLogContext_t& ctx, const GameAction* action)
+    {
+        MemoryStream& output = ctx.output;
+
+        char temp[128] = {};
+        snprintf(temp, sizeof(temp), "[%s] Game Action %08X (", GetRealm(), action->GetType());
```

Comment:
Game action should have a friendly name here, e.g. the enum name.

Reply:
I felt like not creating a list at this point since we are still using the old game command ids, once we are past the old game commands its much easier to make this dynamic where no one needs to worry about numbers.